packages/installer/deploy.sh

```diff
@@ -89,6 +93,8 @@ mount_image() {
         mount_state
     fi
 
+    is_mounted /usr/local || mount ${PERSISTENT} /usr/local
```

Comment:
upgrade script should also do this?

Reply:
cos-upgrade is running from a system which has the partitions already in place, so `/usr/local` should be already mounted by dracut on boot